### PR TITLE
add "Restore Old Reblogs" extension

### PR DESCRIPTION
**Pull request by ewenig**
This extension restores the look + feel of the old-style post reblogs on Tumblr as best as possible.

**Advantages:**
* Tested working with every type of post 
* Preserves the title of a post
* Plays nicely with version 2.0 of Editable Reblogs
* Fully cleans up after itself in the extension destructor.

**Disadvantages:**
* Might break things for people who don't have the update, I can't test for that case.
* No extension icon yet.
* The name's kind of clunky, I'm open to suggestions.

**Screenshots:**
*without Restore Old Reblogs*

*with Restore Old Reblogs*

### CODE REVIEW

File: Extensions/old_reblogs.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+//* TITLE Restore Old Reblogs **//
+//* VERSION 1.2.1 **//
+//* DESCRIPTION	Restores the old reblogging style on the Dashboard **//
+//* DEVELOPER eli <eli@csh.rit.edu> (forked from Editable Reblogs by dlmarquis) **//
```
Comment (ThePsionic) [inline Extensions/old_reblogs.js]: XKit automatically makes a link in the control panel to github.com/[developer]. You might want to just input your GitHub username here.

**Comment by ThePsionic** [comment]
LGTM, just the small nitpick about the developer field.

**Comment by Wolvan** [comment]
This might have the same problems as #421, it probably should be a tweak than a seperate extension
/cc @nightpool

**Comment by nightpool** [comment]
going to merge this, #419 and #421 into a single new extension and then push that.

**Comment by ewenig** [comment]
Thanks! I'm going to keep making tweaks to this on my local branch as I find edge cases that break with this enabled. @nightpool please tag me into the PR when it's finished so I can review my part of the code.

**Comment by nightpool** [comment]
will do

**Pull request closed without merging**